src/uct/sm/mm/xpmem/configure.m4

```diff
@@ -51,7 +51,10 @@ AS_IF([test "x$xpmem_happy" = "xno" -a -d "$with_xpmem"],
       ]
      )
 
-AS_IF([test "x$xpmem_happy" = "xyes"], [uct_modules="${uct_modules}:xpmem"])
+AS_IF([test "x$xpmem_happy" = "xyes"],
+      [uct_modules="${uct_modules}:xpmem"
+       uct_test_modules+=" xpmem"]
+       uct_static_modules+=" ucx-xpmem")
```

Comment:
why not use uct_test_modules?

Reply:
> why not use uct_test_modules?

it is different lists: as per offline discussion CMA & KNEM tl's are not tested for now due to missing detection of ptrace permissions in script, but it is used to link and may be tested manually during debug

Reply:
maybe take the list of transports from the output of non-static ucx_info? e.g take a hard-coded list (in test script) of transports that support static link, and filter them by ucx_info output

Reply:
> maybe take the list of transports from the output of non-static ucx_info? e.g take a hard-coded list (in test script) of transports that support static link, and filter them by ucx_info output

used hardcoded list + command line to set extra tls to test